app/models/invenio_rdm_record_converter.rb

```diff
@@ -90,6 +90,8 @@ def invenio_metadata(gf)
       "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} },
       "description": gf.description.first,
       "additional_descriptions": gf.description.last(gf.description.size-1).map{ |add_desc| {"description": add_desc, "type": "other", "lang": "eng"} },
+      "publisher": gf.publisher.shift,
```

Comment:
If no publisher present, `gf.publisher.shift` returns `nil` which responds to `to_json` with `"null"`. We can change this to something else if we want. Maybe an empty string?